types/src/front/lib/connectors_api.ts

```diff
@@ -104,6 +105,7 @@ export type ContentNode = {
   permission: ConnectorPermission;
   dustDocumentId: string | null;
   lastUpdatedAt: number | null;
+  dustTableId: string | null;
```

Comment:
Can we not add this and instead rely on a function in type named
`dustTableIdFromContentNodeInternalId` ? 

I'd rather have a dirty implementation in that function than add a field to this API.

Reply:
That is fair

Reply:
Also not sure why we need it since we got away with it so far?

Reply:
Well we need it to create the tables query config from a content node :) but can def be a function in front

Reply:
Didn't we do that already?

Reply:
We were pulling a list of tables from core so we had table IDs. Now we’re pulling a list of content nodes

Reply:
+1 with @spolu if we can avoid cluttering `ContentNode` type with `dustTableId` and infer it from the internal id?

Reply:
Makes sense :+1:

Reply:
Ok, did that. The downside is that we don't really know if the table exists (for notion, all databases aren't necessarily already processed as structured data since we never backfilled).

I'll either do a backfill, or figure out a way to not return the databases that are not already uploaded as structured data in the `tables` view